presto-geo/src/test/java/com/facebook/presto/geo/TestGeoQueries.java

```diff
@@ -0,0 +1,281 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.facebook.presto.geo;
+
+import com.facebook.presto.Session;
+import com.facebook.presto.metadata.FunctionFactory;
+import com.facebook.presto.testing.LocalQueryRunner;
+import com.facebook.presto.tests.AbstractTestQueryFramework;
+import com.facebook.presto.tpch.TpchConnectorFactory;
+import com.google.common.collect.ImmutableMap;
+import com.google.common.collect.Iterables;
+import org.testng.annotations.Test;
+
+import static com.facebook.presto.testing.TestingSession.testSessionBuilder;
+import static com.facebook.presto.tpch.TpchMetadata.TINY_SCHEMA_NAME;
+
+public class TestGeoQueries
+        extends AbstractTestQueryFramework
+{
+    public TestGeoQueries()
+    {
+        super(createLocalQueryRunner());
+    }
+
+    @Test
+    public void testSTPolygon()
+            throws Exception
+    {
+        assertQuery("select st_polygon('polygon ((1 1, 1 4, 4 4, 4 1))')", "select 'POLYGON ((1 1, 4 1, 4 4, 1 4, 1 1))'");
```

Comment:
It might be obvious to people with domain knowledge why these are in different orders, but it isn't to me. I'm assuming that it's correct, but it's the sort of thing having a link to some background documentation in the docs could clear up.